code/__DEFINES/items.dm

```diff
@@ -111,4 +111,8 @@
 					/obj/spawner/exosuit_equipment = 3,\
 					/obj/spawner/cloth/holster = 4,\
 					/obj/item/stash_spawner = 4,\
-					/obj/item/weapon/storage/deferred/crate/german_uniform = 4)
+					/obj/item/weapon/storage/deferred/crate/german_uniform = 4,\
```

Comment:
ahm, this code only exists because the biomes have not been fully implemented yet. and it will be removed soon. In its accomplishment, you could label them as boxes, SPAWN_TAG_BOX.

Reply:
Here's the issue - these don't really fit into the 'boxes' category. Boxes don't really spawn anywhere, and usually they're supposed to contain mundane stuff like latex gloves and prescription glasses.

Making an entirely new spawn table and spawn rates just for a few crates is out of the scope of this PR. I'd rather build on what we have now, and then once you actually implement loot biomes, we can change it to accomodate that.

Reply:
then tag them appropriately, create a new spawner and add that spawner here.

Reply:
nah